lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/reuse-existing-identity.yaml

```diff
@@ -138,6 +138,8 @@ states:
         targetState: UPDATE_NAME_DOB_PAGE
       cancel:
         targetState: RETURN_TO_RP
+      back:
+        targetState: IDENTITY_REUSE_PAGE_COI
```

Comment:
I think the user could also have come from a some other places e.g. https://github.com/govuk-one-login/ipv-core-back/blob/706eb2d215ba45e5f238a1723d3abdd5082cd64e/lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/update-name.yaml#L67 

Is there a way to determine that we should go back to a different place?

Reply:
Looks like they all are as a result of clicking a 'go back' option themselves so its probably ok for the 'back' here to go back to IDENTITY_REUSE_PAGE_COI